superdesk/io/feed_parsers/__init__.py

```diff
@@ -63,13 +63,17 @@ def set_dateline(self, item, city=None, text=None):
         :type text: str
         """
 
-        item['dateline'] = {}
+        if 'dateline' not in item:
+            item['dateline'] = {}
```

Comment:
Well, I'm not actually back, I just dropped by to say hi and wish you guys a (late) happy new year. :wink: